arch/arm/src/Makefile

```
@@ -168,7 +168,7 @@ ifneq ($(CONFIG_ALLSYMS),y)
 		$(LDSTARTGROUP) $(LDLIBS) $(EXTRA_LIBS) $(LDENDGROUP)
 else
 	$(Q) # Link and generate default table
-	$(Q) $(call LINK_ALLSYMS, $^)
+	$(Q) $(if $(wildcard $(shell echo $(NUTTX))),,$(call LINK_ALLSYMS, $^))
```

```suggestion
	$(Q) $(if $(wildcard $(shell echo $(NUTTX))),,$(call LINK_ALLSYMS,$^))
```
Quoting the [Make documentation](https://www.gnu.org/software/make/manual/make.html#Syntax-of-Functions):
> [...] leading spaces cannot appear in the text of the first argument as written

Otherwise that whitespace will end up actually part of the argument itself.

---

Done